apps/labs/next.config.js

```diff
@@ -30,18 +30,56 @@ const nextConfig = {
       },
     ];
   },
-  async rewrites() {
+  async redirects() {
     return [
       {
-        source: '/plausible/js/script.js',
-        destination: 'https://plausible.io/js/script.js',
-      },
-      {
-        source: '/plausible/api/event',
-        destination: 'https://plausible.io/api/event',
+        source: '/about',
+        destination: '/',
+        permanent: true,
       },
     ];
   },
+  async rewrites() {
+    // The conventions of this return value are described in the Next.js docs:
+    // https://nextjs.org/docs/api-reference/next.config.js/rewrites
+    return {
+      afterFiles: [
```

Comment:
Note to reviewer: please double check that the Plausible rewrites on the left are the same here—meaning, rewrites in the old format are the same if they are put under `afterFiles` key in the new format.